### PR TITLE
Allow '+' to be in branch names in merge PR titles

### PR DESCRIPTION
**Pull request by JornVernee**
Allow '+' to be in branch names in merge PR titles. I've also sharpened the error message to include the exact pattern the bot checks for.

---------
### Progress
- [x] Change must not contain extraneous whitespace
- [x] Change must be properly reviewed

### Reviewers
 * [Robin Westberg](https://openjdk.java.net/census#rwestberg) (@rwestberg - **Reviewer**) ⚠️ Review applies to 5ea7abcf51c4662a75f88b34ad261ea78c7c1f0f

### Download
`$ git fetch https://git.openjdk.java.net/skara pull/970/head:pull/970`
`$ git checkout pull/970`

### CODE REVIEW

File: forge/src/main/java/org/openjdk/skara/forge/PullRequestUtils.java [hunk 1]
```diff
@@ -62,8 +62,8 @@ private static Hash commitSquashed(PullRequest pr, Repository localRepo, Hash fi
     private static Hash fetchMergeSource(PullRequest pr, Repository localRepo) throws IOException, CommitFailure {
         var sourceMatcher = mergeSourcePattern.matcher(pr.title());
         if (!sourceMatcher.matches()) {
-            throw new CommitFailure("Could not determine the source for this merge. A Merge PR title must be specified on the format: " +
-                                            "Merge `project`:`branch` to allow verification of the merge contents.");
+            throw new CommitFailure("Could not determine the source for this merge. A Merge PR title must be specified of the format: `" +
```
Comment (rwestberg) [inline forge/src/main/java/org/openjdk/skara/forge/PullRequestUtils.java:65]: Perhaps "specified *in* the format:" would sound better, but I'm not a native speaker. :)

**Review by rwestberg — APPROVED** [review]
Looks good!

File: forge/src/main/java/org/openjdk/skara/forge/PullRequestUtils.java (hunk 1 above)
Comment (JornVernee) [reply forge/src/main/java/org/openjdk/skara/forge/PullRequestUtils.java:65]: Yeah, that sounds nicer (though I think technically both are correct).

**Comment by JornVernee** [comment]
The check is supposedly in progress, but the action run has already completed, so I'll just integrate this now

**Pull request closed without merging**